packages/shared/src/hooks/vote/types.ts

```diff
@@ -92,8 +92,9 @@ export const voteMutationMatcher: UseMutationMatcher<
 
   return (
     status === 'success' &&
-    mutation?.options?.mutationKey?.toString() ===
-      createVoteMutationKey({ entity }).toString()
+    mutation?.options?.mutationKey
```

Comment:
@capJavert I decided to rather check for includes as the feed one itself has something like:
`'post', 'mutation', 'vote', 'my-feed'` where the page one only has: `'post', 'mutation', 'vote'`

This way they both hit <-> both weys